File: Python/Source Code Python Contest/Contest_0_LamQuenHackRank/Bai13_LonNhatNhoNhat.py
from math import *
x = int(input())
y = int(input())
z = int(input())
t = int(input())
so_lon_hon = max([x, y]) 
so_nho_hon = min([z, t])
so_lon_nhat = max([x, y, z])
so_nho_nhat = min([x, y, z, t])
print(so_lon_hon, so_nho_hon, so_lon_nhat, so_nho_nhat, sep = '\n')

File: C++/Source Code C++/LamQuenVoiHackerRank/Bai13_LonNhatNhoNhat.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	int x ,y  , z , t ; cin >> x >> y >> z >> t;
	cout << max(x , y) << endl;
	cout << min(z , t) << endl;
	cout << max({x , y , z}) <<endl;
	cout << min({x , y , z , t}) << endl;
}